### PR TITLE
Update README and add example config

### PR DESCRIPTION
**Pull request by sttts**
Update and fix the instructions.

### CODE REVIEW

File: configs/example-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  name: publisher-config
+data:
+  config: |
+    # the github org or user to publish the new repos to
+    target-org: <your-github-org-or-user>
+
+    # the github issue number in the source repo to publish logs to on errors. You must be
+    # able to write to that issue. So you probably should create it with the bot user.
+    # REMEMBER: do not run the bot with a user that can close arbitrary issues in the
+    #           source repo as that will triger unwanted close events on push.
```
Comment (marun) [inline configs/example-configmap.yaml]: s/triger/trigger/

File: configs/example [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# the Docker repo for the bot image. Should be writable for you.
+DOCKER_REPO = <your-docker-user>/k8s-publishing-bot
```
Comment (marun) [inline configs/example]: s/docker-user/docker-repo/

**Comment by marun** [comment]
A couple of minor nits.

File: configs/example (hunk 2 above)
Comment (sttts) [reply configs/example]: actually it's `<docker-user>`, i.e. `<your-docker-user>/k8s-publishing-bot` is the repo, isn't it?

File: configs/example-configmap.yaml (hunk 1 above)
Comment (sttts) [reply configs/example-configmap.yaml]: fixed

File: configs/example (hunk 2 above)
Comment (marun) [reply configs/example]: nmind, repo says 'server' to me, not 'user'.

**Pull request merged**